frontend/webapp/containers/main/actions/action-drawer-container/index.tsx

```diff
@@ -70,11 +70,9 @@ const ActionDrawer: React.FC<Props> = () => {
   const handleSave = async (newTitle: string) => {
     if (!validateForm()) {
       notify({
-        message: 'Required fields are missing!',
-        title: 'Update Action Error',
         type: 'error',
-        target: 'notification',
-        crdType: 'notification',
+        title: 'Update',
+        message: 'Required fields are missing!',
```

Comment:
can we handle messages and titles in a const file

export const ERROR_MESSAGES = {
    FETCH_FAIL: 'Failed to fetch data. Please try again later.',
    MISSING_FIELDS: 'Required fields are missing!'
    // Add more as needed
};